web/src/components/wizard/tests/ConfigurationStep.test.tsx

```diff
@@ -1507,4 +1509,145 @@ describe.each([
     const enableSslField = screen.getByTestId("bool-input-enable_ssl") as HTMLInputElement;
     expect(enableSslField.checked).toBe(true); // default "1" is used since getEffectiveValue includes default
   });
+
+  describe("Default value utility functions", () => {
+    // Test version of the component using the new HelpText component
+    const TestComponent = () => {
```

Comment:
why create an entire new test component just to test the HelpText component? this test doesn't seem relevant to the ConfigurationStep component.

Reply:
yeah, you're right. i think this is left over from the prior implementation